keyboards/gmmk/pro/ansi/keymaps/mattgauf/keymap.c

```diff
@@ -173,7 +173,8 @@ bool encoder_update_user(uint8_t index, bool clockwise) {
             }
         }
     }
-    return true;
+    //return true; //set to return false to counteract enabled encoder in pro.c
+    return false;
```

Comment:
Acknowledging with approval, I'll be sure to pull the changes into my fork once the PR is complete.